contributing.md

```diff
@@ -125,13 +125,16 @@ learn and become confident about git, like http://try.github.io/.
 
 ### Create a virtual environment for plotly development
 
-You can use either [conda][conda-env] or [virtualenv][virtualenv] to create a virtual environment for plotly development, e.g.
+You can use either [conda][conda-env] or [virtualenv][virtualenv] to create a virtual environment for plotly development, e.g.:
 
 ```bash
-conda create -n plotly-dev python
+conda create -n plotly-dev python=3.11
 conda activate plotly-dev
 ```
 
+Please note that our dependencies require Python 3.11 as of May 2024.
```

Comment:
I think it should still work with Python3.8 - 3.10.

Reply:
I'm testing with Python 3.8-3.10 now…